src/lib/components/Cytoscape.react.js

```diff
@@ -321,6 +321,14 @@ class Cytoscape extends Component {
             setExtent(cy.extent());
         });
 
+        // Refresh layout if current zoom is out of boundaries
+        cy.on('layoutstop', function () {
```

Comment:
When does the `layoutstop` event fire? The docs aren't really clear but my guess would be that it only fires when an element is moved or the elements change -- what have you seen in your testing?

I like this approach but wondering whether `layoutstop` is the right event to attach it to.

Reply:
Good point! `layoutstop` only gets triggered after the layout is rebuilt so it was getting triggered with the callback from the demo but not if the callback just updates leaflet's `children`. I made a custom event that gets triggered every time `maxZoom` or `minZoom` changes and it's now working as expected.